docs/docs/outputs/output-options.md

```diff
@@ -53,8 +53,7 @@ Available options:
 
 5. **sort-events**
 
-    This makes it possible to sort the events as they happened. Especially in systems where Tracee tracks lots of events, it can happen that they are received unordered. More information is provided in the [deep-dive](../deep-dive/ordering-events.md) section of the documentation.
-
+    This makes it possible to sort the events as they happened. Especially in systems where Tracee tracks lots of events, it can happen that they are received unordered.
```

Comment:
It exists: https://github.com/aquasecurity/tracee/blob/main/docs/docs/advanced/ordering-events.md

Reply:
This needs to be handled before merging. 👍🏻